new-lamassu-admin/src/components/ConfirmDialog.js

```diff
@@ -0,0 +1,82 @@
+import { Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle as MuiDialogTitle, IconButton } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
```

Comment:
Check out how we're currently handling styles on Funding.js

We're using a hook instead of this HOC.